clients/rospy/src/rospy/impl/tcpros_service.py

```diff
@@ -154,7 +154,7 @@ def contact_service(resolved_name, timeout=10.0):
             except: # service not actually up
                 if first:
                     first = False
-                    rospy.core.logerr("wait_for_service(%s): failed to contact [%s], will keep trying"%(resolved_name, uri))
+                    rospy.core.logwarn("wait_for_service(%s): failed to contact [%s], will keep trying"%(resolved_name, uri))
```

Comment:
When no timeout is specified the default value is 10s. So the message would get printed repeatedly - not only once. Therefore I would argue both log levels should be the same. Not sure if `info` or `warn` make more sense though.